config/runtimes/kserve-mlserver.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
+metadata:
+  name: kserve-mlserver
+spec:
+  supportedModelTypes:
+    - name: sklearn
+      version: "0"
+    - name: xgboost
+      version: "0"
```

Comment:
xgboost should be version 1 according to https://kserve.github.io/website/modelserving/v1beta1/serving_runtime/

Reply:
Thanks, @yuzisun . I will update version by referring the link.